### PR TITLE
feat: add markdown glamour styles

### PR DESCRIPTION
**Pull request by getinnocuous**
This is fairly standalone and can be merged into main once #237 has.

### CODE REVIEW

**Review by DavidS-ovm — APPROVED** [review]

**Comment by getinnocuous** [comment]
dark-theme
light-theme

File: cmd/theme.go [hunk 1]
```diff
@@ -117,8 +118,13 @@ var deletedLineStyle = lipgloss.NewStyle().Background(lipgloss.Color(ColorPalett
 var containerStyle = lipgloss.NewStyle().PaddingLeft(2).PaddingTop(2)
 
 func markdownToString(markdown string) string {
+	themePath := "./overmind-theme.json"
+	hasDarkBackground := termenv.HasDarkBackground()
+	if hasDarkBackground {
+		themePath = "./overmind-theme-dark.json"
+	}
 	r, _ := glamour.NewTermRenderer(
```
Comment (DavidS-ovm) [inline cmd/theme.go:126]: Here and below is missing error handling.

**Review by DavidS-ovm — CHANGES_REQUESTED** [review]
Using `markdownToString` fails with a segfault. adding the missing error handling reveals

```
open ./overmind-theme.json: no such file or directory
```

This should instead use `embed`, e.g. like https://github.com/overmindtech/cli/blob/12b9c379d62382e7b2848916acb3d1b574ad5e4e/cmd/root.go#L32-L33

**Comment by DavidS-ovm** [comment]
@getinnocuous maybe add a small test for the method so I don't have to run the code in my head?

**Comment by DavidS-ovm** [comment]
I've added the test to give you a leg up here, @getinnocuous .

Before merging this also should be rebased and squashed to clean up the history.

**Review by DavidS-ovm — APPROVED** [review]
Sweet! Please don't forget to rebase or squash this before (or during) merge.

**Pull request merged**